src/index.ts

```
@@ -9,6 +9,7 @@ export {
 export { usePaths } from './composables/usePaths'
 export { useSidebar } from './composables/useSidebar'
 export { createOpenApiSpec, type OpenApiSpecInstance } from './lib/spec/createOpenApiSpec'
+export { markdownToPlainText, type MarkdownToPlainTextOptions } from './lib/markdown/markdownToPlainText'
```

ESLint

[error] Expected "./lib/markdown/markdownToPlainText" to come before "./lib/spec/createOpenApiSpec".

(perfectionist/sort-exports)